_health-care/_js/_components/health-care-app.jsx

```diff
@@ -12,26 +13,51 @@ class HealthCareApp extends React.Component {
   constructor() {
     super();
     this.publishStateChange = this.publishStateChange.bind(this);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
     let initialState = {
       applicationData: {
-        introduction: {},
+        introduction: {
+          introduction: {
+            active: true
```

Comment:
Hmm... so the active location should be the same as the URL I think?

Is it fair to assume that any page should map to a URL so you can have a history of it?  If so, then I think we should be finding a way to query the ReactRouter...for the location:

https://github.com/reactjs/react-router/blob/master/docs/API.md#hashhistory

This probably means we need to embed the router inside an App container or something. But that's a second stage refactor.

For now, what about having a function like `getNextUrl()` which inspects `window.location.fragment` and returns a `<Link>` that you can pass into the ContinueButton via a prop?